tools/build_variables.bzl

```diff
@@ -150,6 +150,7 @@ libtorch_profiler_sources = [
     "torch/csrc/profiler/api.cpp",
     "torch/csrc/profiler/kineto_shim.cpp",
     "torch/csrc/profiler/nvtx_observer.cpp",
+    "torch/csrc/profiler/itt_observer.cpp",
```

Comment:
Hmm, will this compile on non-x86 cpus?

Reply:
Yes, I think so. This file will be compiled on all platforms.
It checks if ITT is enabled in runtime.
  TORCH_CHECK(
      torch::profiler::impl::ittStubs()->enabled(),
      "Can't use ITT profiler - PyTorch was compiled without ITT");